test/test_requirements.py

```diff
@@ -0,0 +1,148 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import csv
+import os
+import re
+import subprocess
+from importlib.metadata import version
+
+REQ_TXT = os.path.abspath(
+    os.path.join(os.path.dirname(__file__), "..", "requirements.txt")
+)
+REQ_CSV = os.path.abspath(
+    os.path.join(os.path.dirname(__file__), "..", "requirements.csv")
+)
+DOC_TXT = os.path.abspath(
+    os.path.join(os.path.dirname(__file__), "..", "doc", "requirements.txt")
+)
+DOC_CSV = os.path.abspath(
+    os.path.join(os.path.dirname(__file__), "..", "doc", "requirements.csv")
+)
+
+SCAN_CSV = os.path.join(
```

Comment:
Just a question, why is everything `os.path.abspath(os.path.join(...))` but this one is `os.path.join(os.path.join(...))`?

This might be more subjective, but maybe do `from os import path` as you write it so many times and 
```python
ROOT_PATH = path.abspath(
    path.join(path.dirname(__file__), "..")
)
```
as you do the same thing 6 times.

Reply:
Ah, I see why, because it has a different combination of parentheses. It works but should be the same as the previous ones probably.

I am not even sure that `abspath` is needed here? I might be missing something, but see how I've changed paths logic a bit [here](https://github.com/Molkree/cve-bin-tool/commit/02f36cb023fe127feabf8ad3f58001257cf9df0c), it looks better imo.

Reply:
This looks much better. Thanks!